app/presenters/summary/sections/employment_details.rb

```diff
@@ -0,0 +1,32 @@
+module Summary
+  module Sections
+    class EmploymentDetails < Sections::BaseSection
+      def name
+        :employment_details
+      end
+
+      def show?
+        income.present? && super
+      end
+
+      def answers
+        [
+          Components::ValueAnswer.new(
+            :lost_job_in_custody, income.lost_job_in_custody,
+            change_path: edit_steps_income_lost_job_in_custody_path
+          ),
+          Components::DateAnswer.new(
+            :date_job_lost, income.date_job_lost,
+            change_path: edit_steps_income_lost_job_in_custody_path
+          ),
+        ].select(&:show?)
+      end
```

Comment:
Could there be a situation where `income.lost_job_in_custody` is `nil`? And then this method would throw an exception for violating the type? Not sure if that is an issue. If it is an issue is a guard statement warranted?

Reply:
Tested by @arthurashman locally. No issues.